.github/workflows/main.yml

```diff
@@ -55,28 +83,17 @@ jobs:
         working-directory: hedera-services
         run: docker build -t services-node:$TAG .
 
-      - name: Set up Go 1.19
-        uses: actions/setup-go@v3
-        with:
-          go-version: 1.19
-        id: go
-
-      - name: Fetch Hedera SDK
-        uses: actions/checkout@v3
-        with:
-          path: hedera-sdk-go
-
       - name: Run Hedera SDK Tests
         env:
-          OPERATOR_KEY: "302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137"
-          OPERATOR_ID: "0.0.2"
+          OPERATOR_KEY: ${{ secrets.LOCAL_OPERATOR_KEY }}
```

Comment:
If a contributor decides to fork the repository and open PR, the secret variables would most likely not be set and the CI checks will fail